src/FakeItEasy/Creation/IFakeOptionsForWrappers.cs

```diff
@@ -1,22 +1,18 @@
 namespace FakeItEasy.Creation
 {
-#if FEATURE_SELF_INITIALIZED_FAKES
     using FakeItEasy.SelfInitializedFakes;
-#endif

```

Comment:
Nice touch. I would've remembered to remove these from the netstd project, but I am not convinced that I would've thought to go after the conditional compilation directives here.